src/js/main.js

```diff
@@ -122,6 +120,30 @@ var SupportKit = Marionette.Object.extend({
             .done();
     },
 
+    logout: function() {
+        // delete the deviceid cookie
+        document.cookie = SK_STORAGE + '=; expires=Thu, 01 Jan 1970 00:00:01 GMT;';
+    },
+
+    checkLocalStorage: function(userId) {
```

Comment:
Is this method only used by getDeviceId? If so, maybe it doesn't deserve to be its own one liner method